fetch.bs

```diff
@@ -1055,6 +1083,9 @@ must be an algorithm accepting an exception.
    <dd>
     <ol>
      <li><p>Let <var>continueAlgorithm</var> be null.
+     <li><p>If <var>timingInfo</var> is not null, and <var>timingInfo</var>'s
+     <a for="fetch timing info">response start time</a> is not zero, set <var>timingInfo</var>'s
+     <a for="fetch timing info">response start time</a> to the result of calling <a for=/>current time</a>.
```

Comment:
Should we base `current time` on https://w3c.github.io/hr-time/#dfn-current-high-resolution-time?

Reply:
No, since `current-high-resolution-time` assumes a `current global` which is not defined for `fetch`.
An alternative would be to define `present time` in the `hr-time` spec, as referenced in `current-high-resolution-time` first bullet, perhaps specify it as `value returned from the global monotonic clock` (or something similar).

Reply:
That would give values that are significantly different from what RT/NT are currently returning. Seems to me we can get to the relevant global object through the request's client, assuming it is not null.

Reply:
We can do that, but I thought instead to use the global monotonic time and adjust to the time origin when marking (which is sort-of how implementations do this).

Reply:
OK, so for that we would need to:
* Expose an "absolute HR time" algorithm in HR-time
* Use that in "current time"
* Run "relative HR time" on the coarsen timestamps before exposing them in RT

It's true that this brings us closer to implementations. I'm also fine with moving "current time" to this newly exposed HR time algorithm as a followup

Reply:
Proposing this for HR-TIME: https://github.com/w3c/hr-time/pull/107
Will amend this patch once we expose the relevant time function from HR-TIME